plugins/obs-ffmpeg/texture-amf.cpp

```diff
@@ -1158,6 +1158,7 @@ static bool amf_avc_init(void *data, obs_data_t *settings)
 				    : 250;
 
 	set_avc_property(enc, IDR_PERIOD, gop_size);
+	set_avc_property(enc, HEADER_INSERTION_SPACING, gop_size);
```

Comment:
This should probably be only conditionally enabled like the implementation for NVENC/x264 is, see PR #3083 for reference.